### PR TITLE
feature/extend_minimum_flutter - update library & sample project to s…

### PR DESCRIPTION
**Pull request by ArkadiGiniApps**
…upport Dart 2.19.6+

### CODE REVIEW

File: .packages
Comment (zeeshan-devrev) [inline .packages:1]: is this file needed @ArkadiGiniApps , it's your local machine's generated one.

Comment (ArkadiGiniApps) [reply .packages:1]: @zeeshan-devrev I think you can remove the file

**Review by zeeshan-devrev — APPROVED** [review]
